### PR TITLE
Fixed bed track position and binding issues.

### PR DESCRIPTION
**Pull request by liana-yang**

### CODE REVIEW

File: src/components/base/view.js [hunk 1]
```diff
@@ -119,6 +119,9 @@ genotet.View.prototype.init = function() {
     .on('genotet.loadFail', function() {
       this.renderer.showFailure();
     }.bind(this));
+
+  // Override jQuery Resizable handle.
+  $('.ui-resizable-handle').css('z-index', 'auto');
```
Comment (yubowenok) [inline src/components/base/view.js]: You are setting z-index for resizable-handles globally. However you should set z-index for handles only in this view.

Note that this method works only when the resizable-handles have a stable DOM order in the view. This is the case for now.

Comment (yubowenok) [inline src/components/base/view.js]: Have you fixed this @Shauro ?

**Comment by yubowenok** [comment]
LGTM

File: src/components/binding/binding-renderer.js [hunk 2]
```diff
@@ -124,9 +128,11 @@ genotet.BindingRenderer.prototype.EXON_LABEL_SIZE = 6;
 genotet.BindingRenderer.prototype.BED_LABEL_SIZE = 13;
 /** @const {number} */
 genotet.BindingRenderer.prototype.BED_MIN_WIDTH = 3;
+/** @const {number} */
+genotet.BindingRenderer.prototype.BED_HEIGHT_PROPORTION = 0.8;
 
 /** @const {!Array<number>} */
-genotet.BindingRenderer.ZOOM_EXTENT = [1, 65536];
+genotet.BindingRenderer.ZOOM_EXTENT = [1, 15673503];
```
Comment (yubowenok) [inline src/components/binding/binding-renderer.js]: Why 15673503?
http://www.wolframalpha.com/input/?i=15673503

Comment (liana-yang) [inline src/components/binding/binding-renderer.js]: I calculated it in this way: 
the range of overview: range = 197194699 - 2999997 = 194194702,
the width of canvas: width = 1239px,
the width of bed label (majority amount): label = 100px,
to guarantee that the labels of neighbor bed rects could be seen clearly, we need the max zoom extent: maxExtent = range \* label / width = 15673503.

Comment (yubowenok) [inline src/components/binding/binding-renderer.js]: This doesn't make sense at all. The range of overview depends on data.

**Pull request merged**